src/controls/tab-bar/tab-bar.js

```diff
@@ -87,20 +81,18 @@ function TabBar({
             role="presentation"
             onKeyDown={(e) => manageFocus(e, { vertical })}
           >
-            <a
+            <button
+              type="button"
               id={'tab-' + toLower(optionValue)} // allow sections to reference tab using `aria-labelledby`
               onClick={() => {
                 onChange(optionValue)
               }}
-              onKeyPress={triggerOnKeys(() => {
```

Comment:
With this being an HTML `<button>` now, are the keyboard commands out-of-the-box? (so this can be removed)

Reply:
Yep, exactly! Trust the platform 🤩